src/App.jsx

```diff
@@ -1,10 +1,24 @@
 import React, { Component } from 'react';
+import { BrowserRouter as Router, Route, Link, Switch } from 'react-router-dom';
```

Comment:
Just import Router and not BrowserRouter so that it enables you to pass history as props

Reply:
@Amoswachira

Reply:
@okay let me work on that

Reply:
@koechkevin I have done that